tasks/customaction.py

```diff
@@ -21,6 +21,7 @@
 # constants
 BIN_PATH = os.path.join(".", "bin", "agent")
 AGENT_TAG = "datadog/agent:master"
+CUSTOM_ACTION_ROOT_DIR = "tools\\windows\\install-help"
```

Comment:
unrelated/todo: there are a few imports up there that are unused we should clean up. We should also lint these tasks in the CI, I think it's the case with a few of them. Can we create a card for this?